### ISSUE TITLE
Cannot read properties of undefined (reading '__globalFiltersGrid')

### ISSUE THREAD

**Issue by dwilches** [RevealBi/reveal-sdk-wrappers#3]
I'm getting this error while testing the beta of web components:

![image](https://github.com/user-attachments/assets/cd58a0f7-9ba8-4424-9873-c9c3c94dc9dd)

As we don't use ids to load dashboards, but we use `loadDashboardFromContainer` I put this in the template of my component:

```
        <rv-reveal-view *ngIf="revealDashboard" [dashboard]="revealDashboard"/>
```

And this in the ngOnInit:

```
RevealApi.RevealSdkSettings.setBaseUrl(myRevealServiceUrl);
RevealApi.RevealSdkSettings.requestWithCredentialsFlag = true;
RevealApi.RevealSdkSettings.setAdditionalHeadersProvider(url => ({ ...}));

defineRevealSdkWrappers();

this.myRestService.getCompanyDashboard("SOME_INTERNAL_ID")
                    .then(rvDashboardBlob => {
                        $.ig.RVDashboard.loadDashboardFromContainer(rvDashboardBlob,
                            revealDashboard => this.revealDashboard = revealDashboard,
                            error => console.error("error occurred", error));
                    });
```

**Comment by brianlagunas** [comment]
There should be no issue using the loadDashboardFromContainer as the wrappers accepts the RVDashboard object for the `dashboard` property.

I would like to mention that things like `defineRevealSdkWrappers();` and `RevealApi.RevealSdkSettings.setBaseUrl(myRevealServiceUrl);` should not happen in ngOnInit. Those should happen much earlier and they only get called once per app.

Is it possible for you to create a very simple example that reproduces the issue?

**Comment by dwilches** [comment]
Hello,

That `ngOnInit` I mentioned is on a different component, a top-level one which is only initialized once, when the Reveal is going to be used for the first time.

I solved this issue by upgrading to Reveal 1.7.0, the wrappers are not working for me with 1.6.1 but work with 1.7.0.

This was the line in 1.6.1 where the issue was coming from:
![image](https://github.com/user-attachments/assets/85c5d7fe-d599-4b89-ac45-ea65941491e1)
